### PR TITLE
Show total on accounts aging report base on filter

### PR DESCRIPTION
**Pull request by apoorv1316**
What
- Show the total on the bottom for selected clients only.

Before
Screenshot 2024-02-26 at 12 51 14 PM

After

Screenshot 2024-02-26 at 12 50 06 PM

### CODE REVIEW

**Review by nisusam — COMMENTED** [review]
`Audit Gems` step is failing in CI

**Comment by apoorv1316** [comment]
@nisusam Please review

File: app/javascript/src/components/Reports/AccountsAgingReport/Container/Table/TableTotal.tsx [hunk 1]
```diff
@@ -2,34 +2,66 @@ import React from "react";
...
+
+      acc.ninety_plus_days += parseFloat(
+        client.amount_overdue.ninety_plus_days
+      );
+      acc.total += parseFloat(client.amount_overdue.total);
+
+      return acc;
+    },
+    {
+      zero_to_thirty_days: 0,
```
Comment (nisusam) [inline app/javascript/src/components/Reports/AccountsAgingReport/Container/Table/TableTotal.tsx:28]: Please update these values to float `0.0`

**Review by nisusam — CHANGES_REQUESTED** [review]

File: app/javascript/src/components/Reports/AccountsAgingReport/Container/Table/TableTotal.tsx (hunk 1 above)
Comment (apoorv1316) [reply app/javascript/src/components/Reports/AccountsAgingReport/Container/Table/TableTotal.tsx]: done

**Review by nisusam — APPROVED** [review]

**Pull request merged**